app/Http/routes.php

```diff
@@ -11,14 +11,17 @@
 |
 */
 
+use \Kaas\Controllers\QuotesController;
+
+
 $app->get('/', function () use ($app) {
     return view('index');
 });
 
 $app->group(['prefix' => 'api', 'middleware' => ['doTheyWantYouToSucceed']], function() use ($app) {
 
     $app->get('quote', function () {
```

Comment:
Can be linked to controller without closure: 

$app->get('user/{id}', 'UserController@show');